contracts/mocks/account/paymaster/PaymasterSignerECDSAMock.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {PackedUserOperation} from "@openzeppelin/contracts/interfaces/draft-IERC4337.sol";
+import {ERC4337Utils} from "@openzeppelin/contracts/account/utils/draft-ERC4337Utils.sol";
+import {PaymasterSigner, EIP712} from "../../../account/paymaster/PaymasterSigner.sol";
+import {SignerECDSA} from "../../../utils/cryptography/SignerECDSA.sol";
+
+contract PaymasterSignerECDSAMock is PaymasterSigner, SignerECDSA {
+    using ERC4337Utils for *;
+
+    constructor(address signerAddr) EIP712("MyPaymasterECDSASigner", "1") {
+        _initializeSigner(signerAddr);
+    }
+
+    // WARNING: No access control
+    function deposit() external payable {
+        _deposit();
+    }
+
+    // WARNING: No access control
+    function addStake(uint32 unstakeDelaySec) external payable {
+        _addStake(unstakeDelaySec);
+    }
```

Comment:
In a mock this is not necessary, the internal function are already available thanks to the hardhat-exposed plugin.

Reply:
Yes, but they're not exposed as payable, so trying to call `$_addStake` and `$_deposit` will fail

Reply:
hooo, that is interresting. Maybe we should change that about the hardhat-exposed plugin

Reply:
please checkout https://github.com/frangio/hardhat-exposed/pull/38

Reply:
Looks good, is there a side effect of making functions payable? I wonder if it could break something on the vanilla repository

Reply:
My guess is that making payable just removes a check that msg.value is 0. Adding a check should not change any behavior. All the calls that are performed without value should not be affected.